packages/cli/src/util/paths.ts

```diff
@@ -31,9 +32,35 @@ export function createExamplePath(
 ): E.Either<Error, string> {
   return DoEither.bind('pathData', generateTemplateAndValuesForPathParams(operation))
     .bindL('queryData', ({ pathData }) => generateTemplateAndValuesForQueryParams(pathData.template, operation))
-    .return(({ pathData, queryData }) =>
-      URI.expand(queryData.template, transformValues({ ...pathData.values, ...queryData.values }))
-    );
+    .return(({ pathData, queryData }) => {
+      // replace "-" in template path and query params
+      const cleanedTemplate = queryData.template.replace(/(?<=\{.*)(?=.*\})(-)/g, '');
+      const realValues = transformValues({ ...pathData.values, ...queryData.values });
+      const cleanedValues = {};
+      for (const realValue in realValues) {
+        const cleanedValue = realValue.replace(/-/g, '');
+        cleanedValues[cleanedValue] = realValues[realValue];
+      }
```

Comment:
I can see this is the same logic that is used in the validator file. I suggest to encapsulate this in a function and export it from the package so we have a single point to modify in the future, if required.

Reply:
It is similar but not the same.

The one in Validator file is going from cleaned (no hyphens) --> real (with hyphens).

This one is going from real values (with hyphens) --> cleaned (no hyphens).

But I will update this to not use `for... in` syntax too, since you requested that for the Validator part, and instead build the object from the array using the reduce syntax. Even though I think it's way less readable.

Reply:
yeah we can still abstract it to a method... duh 🤦‍♀️ . I'll do that.

Reply:
haha nvm again. The things we're iterating over to build the object are of different types. I don't think it's worth to try to make one function that works for both types, so I'm not going to do that. If you feel really strongly about doing this, let's discuss.